manifest-list.md

```diff
@@ -16,7 +16,7 @@ This section defines the `application/vnd.oci.image.manifest.list.v1+json` [medi
 
   This REQUIRED property contains the media type of the manifest list.
   For this version of the specification, this MUST be set to `application/vnd.oci.image.manifest.list.v1+json`.
-  For the media type(s) that this is compatible with see the [matrix](media-types.md#compatibility-matrix).
+  For the media type(s) that this specification is compatible with, see the [matrix](media-types.md#compatibility-matrix).
```

Comment:
I'm fine with the new comma, but against the “specification” insertion.  In this sentence, “this” is referring to `application/vnd.oci.image.manifest.list.v1+json` from the previous sentence.  If the “this” in this sentence is unclear, I'd rather replace it with the media type:

> For the media type(s) that `application/vnd.oci.image.manifest.list.v1+json` is compatible with, see the [compatibility matrix](media-types.md#compatibility-matrix).